compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/selection/TextFieldSelectionManager.kt

```diff
@@ -201,10 +199,7 @@ internal class TextFieldSelectionManager(val undoManager: UndoManager? = null) {
      */
     internal var latestSelection: TextRange? = null
 
```

Comment:
These are the common changes. Are they critical? See https://docs.google.com/document/d/15vffH9b_8Jqy4I9RK103HgCwr1c8lhhm1MGT7pvfT8w/edit?tab=t.0 for the poliucy

Reply:
I'll try to create a CL next week if nothing urgent happens. 

But we need this to be merged.